feat[gen2]: content inputs changes should trigger updates in editor (#3882)
BuilderIO/builder: .changeset/breezy-zebras-hear.md

---
"@builder.io/sdk-react": patch
"@builder.io/sdk-react-native": patch
"@builder.io/sdk-solid": patch
"@builder.io/sdk-svelte": patch
"@builder.io/sdk-vue": patch
"@builder.io/sdk-angular": patch
"@builder.io/sdk-react-nextjs": patch
"@builder.io/sdk-qwik": patch
---

> fix: changes to content state via inputs are not triggering updates in the editor

Reviewer: ```suggestion
Fix: Updating input values while in the Content Editor not triggering changes in the iframe.
```
